auraed/src/runtime/cells/cells.rs

```diff
@@ -30,13 +30,16 @@
 
 use super::{Cell, CellName, CellsError, Result};
 use std::{collections::HashMap, sync::Arc};
-use tokio::sync::Mutex;
+use tokio::sync::{Mutex, MutexGuard};
 
-/// CellsTable is the in-memory store for the list of cells created with Aurae.
+// TODO (future-highway): would a RWLock be more performant?
```

Comment:
i don't think we have read-only access (yet?) so Mutex is likely fine.

Reply:
though if we have a way to dump out the cache from somewhere for logging (making this up) then we might want it. but even then i don't think Mutex here is going to be any sort of performance overhead.

Reply:
I agree. The more I think about it, the less I'm thinking we would benefit from RWLock; there should be a note in the next line, I just wasn't ready to remove the TODO.